frontend/src/components/table/Table.js

```diff
@@ -452,7 +452,13 @@ export default class Table extends PureComponent {
   renderEmptyInfo = (rows) => {
     const { emptyText, emptyHint, pending } = this.props;
     if (pending) {
-      return <div>LOADING...</div>;
+      return (
+        <div className="spinner-wrapper-in-tab">
+          <div>
```

Comment:
why so many divs?

Reply:
because it needs to be wrapped and positioned correctly within the table

Reply:
Q: couldn't this be done inside the SpinnerOverlay? I see that there we have already handled the case of "modal" and applied the right CSS class. 
Why this case is different?

Reply:
Not quite as we will end up with something like this:
<img width="1656" alt="Screenshot 2022-01-20 at 11 58 08" src="https://user-images.githubusercontent.com/1708561/150316227-441b462b-e69b-4d64-93b8-e051f6464365.png">
This is different because we need a wrapper that will do the positioning of the spinner ( Imagine I need to put the spinner on the right of a div, in center or whatever, within a tooltip . The spinner needs to keep its size only his position relative to the surrounding items will change).


We do not need to embed logic specific of the Tab within the SpinnerOverlay.
<img width="1656" alt="Screenshot 2022-01-20 at 12 04 36" src="https://user-images.githubusercontent.com/1708561/150317420-30c10f67-9c5f-43e1-b7dd-e40c80c22e5c.png">

Reply:
ok, understand